### PR TITLE
feature_release update:266 reimplement update script and fix some errors on the fly

### PR DESCRIPTION
**Pull request by daniwe4**
- reimplement update script
- update xdebug states
- update salt:restart command
- add flag to run uninstall quiet

https://github.com/conceptsandtraining/doil/issues/266

### CODE REVIEW

File: setup/updates/update-20230116.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  doil_system_setup_access
+  if [[ $? -ne 0 ]]
+  then
+    doil_status_failed
+    exit
+  fi
+  doil_status_okay
+
+  doil_status_send_message "Reinstalling salt service"
+  doil_system_replace_salt_stack
```
Comment (tbongers-cat) [inline setup/updates/update-20230116.sh:43]: This deletes _/usr/local/share/doil/setup/*_ and copies _/setup/stack/*_ to that directory a second time, after the copy part has already been done in _doil_system_copy_doil_ (without deleting files first). Wouldn't it be easier to include the deletion in _doil_system_copy_doil_ (if the directory exists) and only copy files once? This way we would also make sure that _doil_system_setup_access_ is done correctly for these files.
Or am I missing something essential here?

**Review by tbongers-cat — COMMENTED** [review]
Looks good to me, with one question concerning file handling (see comment).

File: setup/updates/update-20230116.sh (hunk 1 above)
Comment (daniwe4) [reply setup/updates/update-20230116.sh]: Good hint. I place the deletion into doil_system_remove_old_version. Done!

**Review by tbongers-cat — APPROVED** [review]
Looks great now :)

**Pull request merged**